File: 프로그래머스/0/181841. 꼬리 문자열/꼬리 문자열.java
class Solution {
    public String solution(String[] str_list, String ex) {
        String answer = "";
        
        for(int i = 0; i < str_list.length; i++) {
            String str = str_list[i];
            answer += str.indexOf(ex) != -1 ? "" : str;
        }
        return answer;
    }
}

File: 프로그래머스/0/181841. 꼬리 문자열/꼬리 문자열.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* str_list[], size_t str_list_len, const char* ex) {
    char* answer = (char*)malloc(100 * sizeof(char));
    int idx = 0;
    
    for(int i = 0; i < str_list_len; i++){
        if(strstr(str_list[i], ex) == NULL){
            for(int j = 0; str_list[i][j] != '\0'; j++){
                answer[idx++] = str_list[i][j];
            }
        }
    }
    
    answer[idx] = '\0';
    
    return answer;
}